mobile/src/navigation/root/root.navigation.tsx

```diff
@@ -17,11 +17,11 @@ const screenOptions: NativeStackNavigationOptions = {
 };
 
 const Root: FC = () => {
-  const currentUser = useSelector((state: RootState) => state.auth.user);
+  const hasUser = useAppSelector(({ auth }) => auth);
```

Comment:
It\`s not correct naming bcos auth also includes `dataStatus` key. That`s why auth.user with wrapping Boolean

Reply:
Sorry, I was testing smth, and forgot to change the value